digitalpy/core/telemetry/impl/opentel_tracer.py

```diff
@@ -1,5 +1,5 @@
 from opentelemetry.trace import Tracer as OpenTelTracerImpl
-from digitalpy.telemetry.tracer import Tracer
+from digitalpy.core.telemetry.tracer import Tracer
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>I003 isort expected 1 blank line in imports, found 0